.github/workflows/ci.yml

```diff
@@ -173,17 +151,26 @@ jobs:
         image: rabbitmq:latest
         ports:
           - 5672:5672
-        options: --health-cmd "rabbitmqctl node_health_check" --health-interval 10s --health-timeout 5s --health-retries 5
+        options: >-
```

Comment:
Is this wrap around for white space reasons?

Reply:
yea.  Just making things more readable as well as consistent for each declared service.

Reply:
Sure, that makes sense to me, it just looked weird on the diff! Thank you